imagebuildah/build.go

```diff
@@ -1116,6 +1117,16 @@ func (b *Executor) Commit(ctx context.Context, ib *imagebuilder.Builder, created
 	b.builder.SetEntrypoint(config.Entrypoint)
 	b.builder.SetShell(config.Shell)
 	b.builder.SetStopSignal(config.StopSignal)
+	b.builder.SetHealthcheck(nil)
```

Comment:
Kind of a nit, but why not make this the else of the 
```
if config.Healthcheck != nil
else 
b.builder.SetHealthcheck(nil)
```

Reply:
Sure.  Changed it.